scipy-1.0/paper.tex

```diff
@@ -801,24 +801,31 @@ \subsection*{Language choices}
 
 Rounding out the top three languages in SciPy is C, which is also extremely
 well-established over several decades\cite{Kernighan:1988:CPL:576122} of
-scientific computing. Alongside its objected-oriented relative C++, C can be
-leveraged in Python after being exposed using the Cython language. Cython has
```

Comment:
This bit about Cython was in the middle of the paragraph about C. Since C is so much more prevalent than Cython, I think it deserves its own paragraph.

Reply:
A lot of C is vendored, while a lot of the Cython is developer contributed in recent years, so I think the argument about relative importance can go either way.